HelperMethods.rb

```diff
@@ -1,12 +1,34 @@
 module HelperMethods
 
+  require_relative "users"
```

Comment:
'require's should be out of module/class areas
put them on first file lines please